packages/wdio-devtools-service/src/auditor.js

```diff
@@ -94,22 +94,25 @@ export default class Auditor {
     }
 
     async getMetrics () {
-        const timeToFirstByte = await this._audit(TTFBMetric, { URL: this.url })
+        const serverResponseTime = await this._audit(ServerResponseTime, { URL: this.url })
         const result = await this._audit(Metrics)
         const metrics = result.details.items[0] || {}
         return {
             estimatedInputLatency: metrics.estimatedInputLatency,
-            timeToFirstByte: Math.round(timeToFirstByte.numericValue, 10),
```

Comment:
Can we keep the `timeToFirstByte` to stay backwards compatible? Let's add a comment to let people know why we have this value twice.

Reply:
Sure, fixed.